package.json

```diff
@@ -595,14 +595,12 @@
         "@azure/arm-resources-profile-2020-09-01-hybrid": "^2.0.0",
         "@azure/arm-subscriptions": "^5.1.0",
         "@microsoft/vscode-azext-azureutils": "0.4.0-alpha.1",
-        "@microsoft/vscode-azext-utils": "0.4.3-alpha",
+        "@microsoft/vscode-azext-utils": "^0.5.2-alpha.0",
         "buffer": "^6.0.3",
         "jsonc-parser": "^2.2.1",
         "uuidv4": "^6.2.13",
         "vscode-nls": "^5.0.1",
         "vscode-uri": "^3.0.7"
     },
-    "extensionDependencies": [
-        "ms-vscode.azure-account"
-    ]
+    "extensionDependencies": []
```

Comment:
@alexweininger Could you revert this?

Reply:
Yes

Reply:
What was the final decision for handling this anyway? Do I need to start working on the GH Action?

Reply:
I don't think we need it for this release, but if you can whip it out really quickly.

We would want it to edit the `package.json`'s extension dependency and then just `vsce publish --target web`, I think

Reply:
Yeah, I think we should be able to edit that upload script fairly easily.